modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/info/item/CalendarBookingInfoItemFields.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.calendar.web.internal.info.item;
+
+import com.liferay.calendar.model.CalendarBooking;
+import com.liferay.info.field.InfoField;
+import com.liferay.info.field.type.BooleanInfoFieldType;
+import com.liferay.info.field.type.DateInfoFieldType;
+import com.liferay.info.field.type.HTMLInfoFieldType;
+import com.liferay.info.field.type.TextInfoFieldType;
+import com.liferay.info.localized.InfoLocalizedValue;
+
+/**
+ * @author Eudaldo Alonso
+ */
+public class CalendarBookingInfoItemFields {
```

Comment:
OPTIONAL/QUESTION

This file was moved but seems that git does not get that way.

This is intentional?

Reply:
The move was intentional; the class was in the incorrect package.

Reply:
I was more to be aware that it feels more that 
deleted and created
than moved (github dixit)